students/11th/changsik/westagram/westagram/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import path, include
+
+urlpatterns = [
+    path('', include('user.urls')),
```

Comment:
앱 이름으로 url을 시작하는 것을 추천드립니다.
```suggestion
    path('user', include('user.urls')),
```